### PR TITLE
Create missing yamls

### PR DESCRIPTION
**Pull request by OdeclasV**

### CODE REVIEW

File: members/SWAU22467_226588.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+bioguide: SWAU22467_226588
+contact_form:
+  driver: phantom
+  steps:
+    - visit: "https://app.leg.wa.gov/pbc/memberEmail/2/0"
+    - fill_in:
```
Comment (tirish) [inline members/SWAU22467_226588.yaml:6]: maybe start it off with a `find`? looks like the page is fully server rendered so probably not strictly necessary, but couldn't hurt

File: members/SWAU22467_226588.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+          value: $SUBJECT
+          required: true
+          options: 
+            max_length: 100
+        - name: message
+          selector: "#Comment"
+          value: $MESSAGE
+          required: true
+          options: 
+            max_length: 1000
```
Comment (tirish) [inline members/SWAU22467_226588.yaml:54]: Didn't we see an issue where we need this to be a bit lower than the true max? Can you test with a message > 1000 chars and ensure this works?

File: members/SWAU22467_226588.yaml (hunk 1 above)
Comment (OdeclasV) [reply members/SWAU22467_226588.yaml:8]: added it

File: members/SWAU22467_226588.yaml (hunk 2 above)
Comment (OdeclasV) [reply members/SWAU22467_226588.yaml]: lowered this to 900 chars

**Review by tirish — APPROVED** [review]

**Pull request merged**